packages/vite/src/node/plugins/css.ts

```diff
@@ -1748,15 +1748,31 @@ const UrlRewritePostcssPlugin: PostCSS.PluginCreator<{
           const replacerForDeclaration = (rawUrl: string) => {
             return opts.replacer(rawUrl, importer)
           }
-          const rewriterToUse = isCssImageSet
+          let rewriterToUse = isCssImageSet
```

Comment:
May I suggest something like the following for better readability? 

```suggestion
         promises.push(
            (async () => {
              if (isCssImageSet) {
                declaration.value = await rewriteCssImageSet(
                  declaration.value,
                  replacerForDeclaration,
                )
              }

              if (isCssUrl) {
                declaration.value = await rewriteCssUrls(
                  declaration.value,
                  replacerForDeclaration,
                )
              }
            })(),
          )
```


or without IIFE:

```suggestion 
if (isCssUrl && isCssImageSet) {
            promises.push(
              rewriteCssImageSet(declaration.value, replacerForDeclaration)
                .then((url) => rewriteCssUrls(url, replacerForDeclaration))
                .then((url) => {
                  declaration.value = url
                }),
            )
          } else {
            const rewriterToUse = isCssImageSet
              ? rewriteCssImageSet
              : rewriteCssUrls
            promises.push(
              rewriterToUse(declaration.value, replacerForDeclaration).then(
                (url) => {
                  declaration.value = url
                },
              ),
            )
          }
```

Reply:
The first way of writing seems to work. The second way of writing will still call `rewriteCssUrls` if both `isCssUrl` and `isCssImageSet` are false.

Reply:
Thanks for your suggestion, the second way of writing it looks better.